CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## v0.19.3
+
+### Changes
+
+- **Breaking change**: Changed the `go.operator-sdk.io` plugin to only write a `plugins` PROJECT field and run the OLM integration plugin if the project version is "3-alpha" or above. ([#3716](https://github.com/operator-framework/operator-sdk/pull/3716))
```

Comment:
Just noticed this.

@estroz Is this really a breaking change?

It's in a patch release where we should not make breaking changes.

Reply:
I discussed this with @estroz ahead of time. The change removes a behavior that never worked to begin with so I wouldn't call it a breaking change. But, since we've already cut 1.0, there is already a release note for this https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md#changes (the PR is # 3697). 

Since this is a borderline case, and it seemed like having it classified as breaking in 1.0 but non-breaking in 0.19.3 would be more confusing, I opted to leave as-is.